examples/add-receiver-creator/rendered_manifests/daemonset.yaml

```
@@ -22,11 +23,13 @@ spec:
   selector:
     matchLabels:
       app: splunk-otel-collector
+      # component: otel-collector-agent - needs to be added when label names are standardised
```

The last ask, can you please migrate it to the helm comment so it's not rendered

```suggestion
      {{/* component: otel-collector-agent - needs to be added when label names are standardised */}}
```

---

I put it in a wrong place, sorry. Also better to use `{{/*-` to avoid empty lines